### PR TITLE
Orion 15.16.0

### PR DESCRIPTION
**Pull request by TalDubovKaltura**
This change is [Reviewable](https://reviewable.io/reviews/kaltura/server/9161)

### CODE REVIEW

**Comment by MosheMaorKaltura** [comment]
Please close this PR as we do not merge directly into the main branch but user dev branches

**Comment by TalDubovKaltura** [comment]
Sure. Done.

File: deployment/updates/scripts/xml/notifications/entryClipJobStartedHttpNotificationTemplate.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yossipapi) [inline deployment/updates/scripts/xml/notifications/entryClipJobStartedHttpNotificationTemplate.xml]: @TalDubovKaltura FYI for next time the file name format should be 
entryClipJobStartedHttpNotification.template.xml and not
entryClipJobStartedHttpNotificationTemplate.xml

Comment (TalDubovKaltura) [reply deployment/updates/scripts/xml/notifications/entryClipJobStartedHttpNotificationTemplate.xml]: @yossipapi Right, so next time I will add a pull request I will change this too

**Pull request closed without merging**